contrib/blacklist/libexec/blacklistd-helper

```diff
@@ -81,7 +81,7 @@ add)
 		    /sbin/pfctl -a "$2/$6" -f -
 		# insert $ip/$mask into per-protocol/port anchored table
 		/sbin/pfctl -qa "$2/$6" -t "port$6" -T add "$addr/$mask" && \
-		    /sbin/pfctl -q -k $addr && echo OK
+		    /sbin/pfctl -qk "$addr" && echo OK
```

Comment:
All else being equal I like `-q -k` better, but even more than that I like avoiding diffs with upstream.

Reply:
😂I had that exact same thought on the change below, but @zoulasc knows better.

Reply:
he does not, he just followed suit with the rest of the code that combined options. he can change it.

Reply:
No worries. Thank you, we’ll abide!